source/docs/casper/resources/tutorials/beginner/counter/commands.md

```diff
@@ -34,7 +34,12 @@ casper-client query-global-state \
 
 This command allows you to query the state of a Casper network at a given moment in time, which is specified by the _state-root-hash_ described above.
 
--   The _node-address_ is the server (localhost when running on a local NCTL network).
+-   The _node-address_ is the server's _IP:PORT_, where port is usually the default RPC-port when setting up an NCTL network is _11101_ (IP == localhost when running on a local NCTL network).
+
+Example:
+```rust
+  casper-client ... --node-address http://localhost:11101
+```
```

Comment:
I like that you introduced example URL for this tutorial :+1: However, I would make it inline to keep bullet list concise:
```suggestion
-   The _node-address_ is the location of the RPC endpoint, which is typically represented in the form `http://IP:PORT`. In this particular tutorial, for a default-configured NCTL network running locally, the address you can use is `http://localhost:11101`.